example/src/SdkMethodSections/DeviceIDSection.tsx

```diff
@@ -1,20 +1,32 @@
 import * as React from 'react';
-import { Button } from 'react-native-elements';
-import { useAmplitude } from '../utils/amplitude';
-
+import { Button, Input } from 'react-native-elements';
 import { SdkSectionLayout } from './SdkSectionLayout';
+import { useAmplitude } from '../utils/amplitude';
 
 export const DeviceIDSection = () => {
   const amplitude = useAmplitude();
+  const [currentDeviceId, setCurrentDeviceId] = React.useState('');
+  const [loading, updateLoading] = React.useState(false);
+
+  const handleOnPress = () => {
```

Comment:
Can we give this a more contextual name like `handleOnPressRegenerateDeviceId` or `handleRegenerateDeviceId`